aea/configurations/loader.py

```diff
@@ -329,7 +331,7 @@ def load_component_configuration(
 
 
 def _load_configuration_object(
-    component_type: ComponentType, directory: Path
+    package_type: PackageType, directory: Path
 ) -> ComponentConfiguration:
```

Comment:
note, you return a ComponentConfiguration but pass a package_type, that's inconcsistent!